webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageConstants.js

```diff
@@ -0,0 +1,9 @@
+import URI from 'urijs';
+import { foremanUrl } from '../../../../foreman_tools';
+
+export const docUrl = (foremanVersion) => {
+  const rootUrl = `https://docs.theforeman.org/${foremanVersion}/`
+  const section = 'Managing_Hosts/index-foreman-el.html#registering-a-host_managing-hosts'
+
+  return(foremanUrl(`/links/manual?root_url=${URI.encode(rootUrl)}&section=${section}`))
```

Comment:
I don't think this will work because I think the `#` is never sent to server. I think you'll need to encode the section too.

Reply:
You could leverage URI.js to generate the whole url without worrying about the encoding, something like:
```javascript
const url = new URI({path: "/links/manual", query: { root_url: rootUrl, section: section }});
return foremanUrl(url.href());
```

Reply:
I wish foremanUrl had a `query` parameter and would leverage `URI.js`, but since this should be cherry picked it's not a good time to introduce it now.